projects/demo/src/modules/markup/skeleton/skeleton.template.html

```diff
@@ -51,7 +51,7 @@
         >
             <strong>Classes</strong>
             are included into global import
-            <code>@taiga-ui/core/styles/taiga-ui-global</code>
+            <code>@taiga-ui/styles/taiga-ui-global</code>
             . You do not need to import anything, just use them
```

Comment:
That is, if you added the styles package. We need to make sure to put some warning in all places like that. Also maybe it makes sense to add a special page about global styles so people will see its structure and maybe only import the parts they need.